lib/httpRequestBuilder.js

```diff
@@ -85,7 +134,7 @@ function requestBuilder (defaults) {
     let req = Buffer.from(baseReq.join('\r\n') + '\r\n\r\n', 'utf8')
 
     if (bodyBuf && bodyBuf.length > 0) {
-      req = Buffer.concat([req, bodyBuf])
+      req = Buffer.concat([req, ...bodyBuf])
```

Comment:
`...` is kind of expensive, we should avoid it.

Reply:
Thanks for teaching me this. an old `unshift` should have been fine but wanted to be cool 😄